### PR TITLE
Several associated shower bug/QOL fixes.

### LINKED ISSUE #46413: Infinite gas by injecting beaker with syringe
Basically #46100 again, but this was fixed for plasma. See #46361.

Still an issue for n2, o2, n2o, co2 when injecting beakers with a syringe.

### LINKED ISSUE #53357: Showers filled with chemicals that release atmos gasses produce way too many moles of gas.
Showers spill chems. Oxygen, nitrogen, nitrous oxide, and carbon dioxide (the chemicals, not the gasses), produce a shitton of gas when pumped into showers using plumbing, even with just one chem generator.

<b>How to reproduce:</b> Make a shower. Connect it with ducting to a chem generator from the plumbing rcd. Set the chem generator to oxygen. Turn on the shower. Die of overpressure.

<b>Shower Turbine:</b>
shower air megawatt

<b>Shower Oxygen Miner:</b>
gas gas gas

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request
So, I may have made at least one... maybe 1.5 math errors. Namely, the reaction volume on every reagent was the full 200u. This is bad for bitcoin, as it means that you can instantly bottom out most reactions and drown someone in 1000 units of water in a single second! That's bad.

Also! On that note, vapor reaction was also transferring reagents that otherwise helped contribute to the above problem. I've yeeted this in kind, but I'm willing to do some more testing and planning to see if they were specific to the reaction volume fix first, but to be safe vapor is vape-aint.

Lastly: Gas chems were producing way, way, WAY too much gas even once the above two points were occuring. So much gas, that it was capable of filling a 8x10 tile airless room in a timed 1 minute 40 seconds. So, we've slowed the roll of the free reagents being produced by synthesizers, and brought it down to 1/5th of the reaction volume to 1/20th. That sounds like a massive leap, but trust me, you could produce millions of KPA very, VERY quickly without it, but at 1/20 it was actually sort of reasonable.

Edit: Apparently this also fixes #46413 as a result of removing the obj_react from the above gasses!

## Why It's Good For The Game

Fixes #53357. Also, prevents reagent synthesizers and showers to behave EXACTLY like gas miners, but free. Also, prevents 1000s of reagents from being multiplied through the use of showers.

## Changelog
:cl:
fix: Showers no longer have the vapor reaction, as before the changes.
fix: Showers no longer produce ludicrously over-scaled chemical reactions.
balance: Gas chems now produce 1/4th less gas when released into the air as before.
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
you may live for now

**Comment by ATH1909** [comment]
Doesn't hercuri rely on being applied via the vapor method?

**Comment by ATH1909** [comment]
> So, we've slowed the roll of the free reagents being produced by synthesizers, and brought it down to 1/5th of the reaction volume to 1/20th. That sounds like a massive leap, but trust me, you could produce millions of KPA very, VERY quickly without it, but at 1/20 it was actually sort of reasonable.

Uh, instead of impacting plumbing balance like that, why not just remove the reaction_obj effects of nitrogen, oxygen, n2o, and the like, while keeping their reaction_turf effects? Wouldn't that fix the issue?

**Comment by LemonInTheDark** [comment]
I'd prefer it if spilling reagents like this didn't create a lot of gas in the first place, I shouldn't be able to plasmaflood with a plumbing rcd. I'll test the rate in this pr in a bit.

**Comment by msgerbs** [comment]
Does this also break plasma grenades for example?

**Comment by ArcaneMusic** [comment]
> 
> 
> Does this also break plasma grenades for example?

Toss me the typepath and I'll verify that they work still.

**Comment by msgerbs** [comment]
> 
> 
> > Does this also break plasma grenades for example?
> 
> Toss me the typepath and I'll verify that they work still.

It's not a type path. I mean like a chem grenade with plasma in it to light a fire or something similar.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1268,18 +1255,11 @@
...
 	taste_description = "something unknowable"
 
-/datum/reagent/carbondioxide/expose_obj(obj/exposed_obj, reac_volume)
-	. = ..()
-	if((!exposed_obj) || (!reac_volume))
-		return 0
-	var/temp = holder ? holder.chem_temp : T20C
-	exposed_obj.atmos_spawn_air("co2=[reac_volume/5];TEMP=[temp]")
-
 /datum/reagent/carbondioxide/expose_turf(turf/open/exposed_turf, reac_volume)
```
Comment (LemonInTheDark) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:1258]: expose_turf has gotta call parent apperantly (ha), wonder why...

**Comment by LemonInTheDark** [comment]
Tested, looks good, we can raise the production in the future depending on how things go, but chem really shouldn't be able to make that much gas that quickly

**Comment by LemonInTheDark** [comment]
Fix your conflicts nerd

**Comment by ArcaneMusic** [comment]
> Does this also break plasma grenades for example?

image
Works.

**Comment by LemonInTheDark** [comment]
Bet

**Comment by Winterous** [comment]
Well this is gonna make it harder to repressurise medbay by tossing a beaker full of oxygen onto the floor.

**Pull request merged**